### ISSUE TITLE
Hi scott, getting these errors mate

### ISSUE THREAD

**Issue by arunmmanoharan** [scotthartbti/android_frameworks_base#27]
Reading program jar [/home/audiogod/bs/out/target/common/obj/APPS/StalkExplorer_intermediates/classes.jar]
Reading library jar [/home/audiogod/bs/out/target/common/obj/JAVA_LIBRARIES/core_intermediates/classes.jar]
Reading library jar [/home/audiogod/bs/out/target/common/obj/JAVA_LIBRARIES/core-junit_intermediates/classes.jar]
Reading library jar [/home/audiogod/bs/out/target/common/obj/JAVA_LIBRARIES/ext_intermediates/classes.jar]
Reading library jar [/home/audiogod/bs/out/target/common/obj/JAVA_LIBRARIES/framework_intermediates/classes.jar]
packages/services/Telephony/src/com/android/phone/Ringer.java:41: cannot find symbol
symbol  : class QuietHoursHelper
location: package com.android.internal.util.beanstalk
import com.android.internal.util.beanstalk.QuietHoursHelper;
                                          ^
packages/services/Telephony/src/com/android/phone/Ringer.java:184: cannot find symbol
symbol  : variable QuietHoursHelper
location: class com.android.phone.Ringer
            final boolean inQuietHours = QuietHoursHelper.inQuietHours(
                                         ^
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
2 errors
realmake: **\* [/home/audiogod/bs/out/target/common/obj/APPS/TeleService_intermediates/classes-full-debug.jar] Error 41
realmake: **\* Waiting for unfinished jobs....
Reading library jar [/home/audiogod/bs/out/target/common/obj/JAVA_LIBRARIES/framework2_intermediates/classes.jar]
Preparing output jar [/home/audiogod/bs/out/target/common/obj/APPS/StalkExplorer_intermediates/proguard.classes.jar]
  Copying resources from program jar [/home/audiogod/bs/out/target/common/obj/APPS/StalkExplorer_intermediates/classes.jar]
audiogod@buildbox2:~/bs$

**Comment by scotthartbti** [comment]
Yes, this isn't ready to build yet... Need to update all the other repos
yet...  Planning on doing it today and releasing new version.  In the
meantime, I just pushed the necessary commit to fix this build error to
packages/services/telephony.  Please pull the new commit if u need to build.

**Comment by arunmmanoharan** [comment]
Okay thanks man

**Comment by DroidThug** [comment]
Did you solve it?

Because I'm getting it, at this point of time :(

Branch BS6

Same, Error 41

**Comment by regalstreak** [comment]
@DroidThug xD

**Comment by DroidThug** [comment]
Lol, that was when I was a proper Noob @regalstreak
